tests/box/box.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* bender-tags: ui, box */
+
+import Box from '/ckeditor5/ui/box/box.js';
+import BoxView from '/ckeditor5/ui/box/boxview.js';
+import Model from '/ckeditor5/ui/model.js';
+
+describe( 'Box', () => {
+	let model, box, view;
+
+	beforeEach( () => {
+		model = new Model( {
+			alignRight: true
+		} );
+
+		view = new BoxView( model, view );
+		box = new Box( model, view );
+	} );
+
+	describe( 'constructor', () => {
+		it( 'should binds view#model attributes to the InputText#model', () => {
+			expect( view.model.alignRight ).to.equal( model.alignRight ).to.equal( true );
```

Comment:
Thanks, it's useful.

I've spotted that some of existing tests check only if `view.model` property is equal to `model` property. It might be not safe in case of typo.

E.g: 
```
expect( view.model.someUndefinedProp ).to.equal( model.someUndefinedProp );
```
Assertion is true, but it doesn't check anything. Thats why I've checked also the real value.